src/content/docs/infrastructure/host-integrations/host-integrations-list/lightptd-integration.mdx

```diff
@@ -0,0 +1,152 @@
+* If you'd rather install our infrastructure agent manually, you can follow a tutorial for manual installation for [Linux](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux), [Windows](/docs/infrastructure/install-infrastructure-agent/windows-installation/install-infrastructure-monitoring-agent-windows/), or [macOS](/docs/infrastructure/install-infrastructure-agent/macos-installation/install-infrastructure-monitoring-agent-macos/).
+
+## Integrate Lighttpd with New Relic [#integrate]
+
+Once installed, the infrastructure agent can ingest data from your app and send it to New Relic, but you still need to integrate – or establish a line of communication – between the agent and your app. Once integrated, you can get started with an out of the box monitoring solution for your Lighttpd app. 
+
+<Steps>
+<Step>
+### Manually configure the lighttpd.conf file
+
+To export metrics, please use the following steps on `127.0.0.1:9880/server-status?format=plain`
+
+1. Go to lighttpd config file path
+```
+cd /etc/lighttpd/
+```
+2. Open `lighttpd.conf` file to edit
+```
+sudo nano lighttpd.conf
+```
+3. Add `mod_status` on your Lighttpd servers.
+4. Add `mod_status` inside `server.modules` object.
+5. Add status.status-url = `/server-status` after server.modules
+6. Update `server.document-root` with the lighttpd file path `/var/www/html/lighttpd-webpage`
+7. Update `server.port` on which you are running lighttpd server.port = 9880
+8. Run `127.0.0.1:9880/server-status?auto` or  `127.0.0.1:9880/server-status?format=plain` on the browser to check server related metrics.
+</Step>
+
+
+<Step>
+
+## Nri-Flex configuration
+
+First, follow the [installation steps](https://github.com/newrelic/nri-flex/blob/master/docs/basic-tutorial.md) of nri-flex from [nri-flex repository](https://github.com/newrelic/nri-flex)
+
+Once you have installed the infrastructure agent on your host, nri-flex binary is also installed with it.
+
+1. Create a flex configuration file
+Path:
+```
+/etc/newrelic-infra/integrations.d
+```
+
+2. Create a file with the name `lighttpd-http-config.yml`
+
+Note: A generic file name could be &ltFILENAME&gt-config.yml
+
+3. Use add the following configuration to your `lighttpd-http-config.yml` file. 
+
+Sample configuration file: https://github.com/newrelic/nri-flex/blob/master/examples/lighttpd-http-example.yml
+
+It's important to not that you can replace `base_url` and `url` with your application url (Please refer to url documentation for more understanding) and also change url in the `commands` attribute. You can also convert the metrics format which you get through url into regex and add expressions in regex_matches. EVENT_TYPE is also adjustable. 
```

Comment:
(Please refer to [url documentation](https://github.com/newrelic/nri-flex/blob/master/docs/apis/url.md) for more understanding)